SETUP/UPGRADE.md

```diff
@@ -80,108 +80,15 @@ you want to run and add the following line immediately after the opening
 $maintenance_override = TRUE;
 ```
 
-### Upgrading from release 1.3 or earlier
-Sorry, there's no automated upgrade mechanism. If you post
-to the 'DP Site Code' forum, we might be able to help you.
+### Upgrading from a release before R202009
 
-### Upgrading from release 1.4
-Run the scripts in the following directories in order
-
-* c/SETUP/upgrade/05/
-* c/SETUP/upgrade/06/
-* c/SETUP/upgrade/07/
-* c/SETUP/upgrade/08/
-* c/SETUP/upgrade/09/
-* c/SETUP/upgrade/10/
-* c/SETUP/upgrade/11/
-* c/SETUP/upgrade/12/
-* c/SETUP/upgrade/13/
-* c/SETUP/upgrade/14/
-
-### Upgrading from release 1.5
-Run the scripts in the following directories in order
+If you are upgrading a release before R202009, you must upgrade to R202009
```

Comment:
How are they supposed to do this? Should we mention a tag or something in git for them to get instructions for upgrading to R202009?

Reply:
They're suppose to download and install R202009 first. I will make that clearer and reference the Releases page.

Reply:
I've added a link to the releases page in the message. I didn't add more specific wording because if they are upgrading from a release, they should have either downloaded it from Github, or they are using git and it should be clear based on the tags.